app/src/main/res/values/component_colors.xml

```diff
@@ -280,6 +280,10 @@
   <color name="component_color_audio_fragment_seekbar_progress_color">@color/color_palette_icon_background_secondary_color</color>
   <color name="component_color_audio_fragment_seekbar_color">@color/color_palette_seekbar_progress_background_color</color>
   <!-- Topic Revision Fragment -->
-  <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_revision_background_color</color>
+  <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_fragment_background_color</color>
+  <!-- Topic Practice Fragment -->
+  <color name="component_color_topic_practice_background_color">@color/color_palette_topic_fragment_background_color</color>
```

Comment:
Replace `topic_practice` with `topic_practice_fragment` in all three values.

Reply:
Okay, actually the background color in palette is common for whole topic that's why I make it common.
But I'll change `topic_practice` with `topic_practice_fragment` in components color.